GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/checkboxes.config

```diff
@@ -209,27 +197,15 @@
     {
       "contentTypeKey": "e0876897-88f4-4dcf-ae0a-ec5a37e3a185",
       "udi": "umb://element/35d9f76c584f4449a9472e5079cbb685",
-      "text": {
-        "markup": "<h2 class=\"govuk-heading-m\">Toggle an option on or off</h2>",
-        "blocks": {
-          "contentData": [],
-          "settingsData": []
-        }
-      }
+      "text": "<h2 class=\"govuk-heading-m\">Toggle an option on or off</h2>"
     },
     {
       "contentTypeKey": "f1cb27f6-c7f5-4db0-bfff-ef76df811ac7",
       "udi": "umb://element/10daa714647e44158595d69ff122c362",
       "conditionalBlocks": "",
       "value": "yes",
       "label": "I agree to something",
-      "hint": {
-        "markup": "<p>You have to agree to this.</p>",
-        "blocks": {
-          "contentData": [],
-          "settingsData": []
-        }
-      }
+      "hint": "<p>You have to agree to this.</p>"
```

Comment:
Changes like this where the text is the same but the format is different indicate your version of uSync is out of date.